openmls/src/group/mls_group/commit_builder.rs

```diff
@@ -265,11 +265,40 @@ impl<'a> CommitBuilder<'a, LoadedPsks> {
     /// Validates the inputs and builds the commit. The last argument `f` is a function that lets
     /// the caller filter the proposals that are considered for inclusion. This provides a way for
     /// the application to enforce custom policies in the creation of commits.
-    pub fn build(
+    pub fn build<S: Signer>(
```

Comment:
Is there a reason to make this a generic? Looks a little inconsistent now with the other types.